src/nvim/drawscreen.c

```diff
@@ -778,10 +778,17 @@ static void win_redr_border(win_T *wp)
     if (adj[3]) {
       grid_put_schar(grid, irow + adj[0], 0, chars[6], attrs[6]);
     }
+
     for (int i = 0; i < icol; i++) {
       int ic = (i == 0 && !adj[3] && chars[6][0]) ? 6 : 5;
       grid_put_schar(grid, irow + adj[0], i + adj[3], chars[ic], attrs[ic]);
     }
+
+    if (wp->w_float_config.footer) {
```

Comment:
@bfredl: I put code here solely based on trial/error and some sense of symmetry. It seems to work, but I might miss something..

Reply:
Symmetry with title is correct here:)